src/api/app/models/branch_package.rb

```diff
@@ -500,32 +500,8 @@ def find_packages_to_branch
 
   def lookup_incident_pkg(p)
```

Comment:
:thought_balloon: I would try to use something a bit more descriptive as variable name

Reply:
Yes, I can do it, to change this method wasn't my goal. Specially that looks like `p` is the thing in this whole `BranchPackage` class